cfy_manager/components/postgresql_server/postgresql_server.py

```diff
@@ -34,11 +36,11 @@
 from ...utils.systemd import systemd
 from ...utils.install import yum_install, yum_remove
 
+SCRIPTS_PATH = join(constants.COMPONENTS_DIR, POSTGRESQL_SERVER, SCRIPTS)
```

Comment:
Maybe POSTGRESQL_SCRIPTS_PATH is a better name

Reply:
Fixed